receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -31,6 +31,13 @@ attributes:
   branch.name:
     description: The name of a Git branch
     type: string
+  pull_request_state:
+    name_override: type
```

Comment:
I'm actually not aware of how this works - does this mean that the attribute name in the emitted metrics will actually be `type` and not `pull_request_state`? Why would we want to have these two different names - `pull_request_state` and `type` and not just use one of those?

Reply:
That's correct, it was showing as `type` in queries. There actually wasn't a good reason for this, and I'm updating to use `pull_request_state` instead.